libs/cardano-ledger-api/cardano-ledger-api.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-api
-version:            1.7.0.1
+version:            1.8.0
```

cardano-ledger-api does indeed need a breaking change, since it re-exports those lenses. This fact needs to be listed in the changelog. Note that it is only the lenses that needs to be listed in the changelog, because types are not re-exported. So only lenses relevant changeloig from cardano-ledger-core and c ardano-ledger-conway need to be listed copied over
```suggestion
version:            1.8.0.0
```